src/fiatExchanges/SelectProvider.test.tsx

```diff
@@ -126,6 +137,7 @@ describe(SelectProviderScreen, () => {
     await waitFor(() => expect(fetchExchanges).toHaveBeenCalledWith('MX', Currency.Dollar))
   })
   it('shows the provider sections, mobile money, and exchange section', async () => {
+    mocked(readOnceFromFirebase).mockResolvedValue(FAKE_APP_ID)
```

Comment:
seems like this is the same mock behavior in every test. best to put it in a `beforeEach` or (probably better) attach it to the jest mock function, e.g. 
```
jest.mock('src/firebase/firebase', () => ({
  readOnceFromFirebase: jest.fn().mockResolvedValue(FAKE_APP_ID),
}))
```